kubejs/server_scripts/enigmatica/kubejs/functions.js

```diff
@@ -76,3 +76,11 @@ const playerHas = (item, player) => {
 function lowerTiers(tiers, tier) {
     return tiers.slice(0, tiers.indexOf(tier));
 }
+
+// transplant the md5 from `<type's mod>:kjs_<md5>` onto the supplied prefix
+function md5(recipe, id_prefix) {
```

Comment:
I strongly dislike the name `md5`, I like function names that say what the function does :P

Reply:
`createRecipeWithIdPrefix` would be fine? 😄

Reply:
hmm, it might push the recipe part too far to the right making it harder to spot when scrolling through 🤔 
![long desk thing](https://i.makeagif.com/media/4-30-2016/xIt2gD.gif)
`renameRecipeIdIfItContainsKjsAndOtherwiseJustKeepTheId(`

Reply:
I just mean the function name, the md5_ part can stay in the recipe id no problem 👍

Reply:
Yea me too, i would still prefer to keep it rather short:

- fallbackId (capital i, generally used for global functions in this repo, but uppercase i is easily confused with L)
- fallback_id (lowercase i, but those might be confused for local variables/functions)
- md5 (was nice and short, hard to confuse with anything else)
- fallback (might sound a bit too generic)
- nameless (honestly sounds fairly confusing)
- createRecipeWithIdPrefix (a bit long and rather vague)

i just can't think of anything descriptive enough that looks good 😅

Reply:
also, the name of the function doesn't _have_ to refer to naming the fallback, it merely catches the output of recipes (and those without a `:kjs_` safely pass through / are ignored), therefore the function could refer to something like:
- `enigmatica(`
- `engmaticRecipe(`
- `save(`
- `register(`
- `store(`
- `process(`

Reply:
Out of all of our options, my favourite is `fallback_id` :D

Reply:
done 👍